cmd/root.go

```diff
@@ -203,9 +209,29 @@ func dateStamp() string {
 	return t.Format("20060102")
 }
 
+var runJobs string
+
 // addJobsToQueue adds the jobs to wr's queue.
 func addJobsToQueue(s *scheduler.Scheduler, jobs []*jobqueue.Job) {
+	if runJobs != "" {
+		testPrint(jobs)
+
+		return
+	}
+
 	if err := s.SubmitJobs(jobs); err != nil {
 		die("failed to add jobs to wr's queue: %s", err)
 	}
 }
+
+func testPrint(jobs []*jobqueue.Job) {
+	const testOutput = 3
```

Comment:
What's the significance of 3? Make this len(jobs) instead?

Reply:
It's the file descriptor used by the tests for reading the jobs. I've renamed the const to make it clearer.